components/kyma-operator/pkg/kymainstallation/step.go

```diff
@@ -123,7 +124,28 @@ func (s upgradeStep) Run() error {
 		releaseOverrides)
 
 	if upgradeErr != nil {
-		return errors.New("Helm upgrade error: " + upgradeErr.Error())
+		upgradeErrMsg := fmt.Sprintf("Helm upgrade error: %s", upgradeErr.Error())
+		errorMsg := upgradeErrMsg
+
+		rollbackWaitTime := 10
+
+		log.Println(fmt.Sprintf("Helm upgrade of %s failed. Performing rollback before retrying upgrade.", s.component.GetReleaseName()))
```

Comment:
Add info about rollback revision -  will be useful for debugging

Reply:
Example log: "Helm upgrade of FOO failed. Rolling back to last known stable revision: X"